src/app/(profile)/(components)/ProfileCard/ProfileCard.tsx

```diff
@@ -70,9 +71,10 @@ const ProfileCard = ({
   const getTechnologyClasses = cx({
     [styles.technology]: true,
   })
+
   return (
     <Link
-      href={href}
+      href={`${AppRoutes.profile}/${data.githubUsername}`}
```

Comment:
This approach would block us getting into moderation view of profile. check #281  and `src\app\(profile)\(components)\ProfileList\ModerationProfilesWithFilter.tsx`